packages/larva-patterns/objects/o-card/o-card.twig

```diff
@@ -0,0 +1,53 @@
+<article class="o-card {{ o_card_classes }}">
+
+	{% if o_card_link_url %}
+		<a tabindex="0" href="{{ o_card_link_url }}" class="{{ o_card_link_classes }}">
+	{% endif %}
+
+	{% if o_card_image_wrap_classes %}
+	<div class="o-card__image-wrap {{ o_card_image_wrap_classes }}">
+		{% endif %}
```

Comment:
Where is this endif coming from? Looks like all if's already have their closing endif, this looks like a floater.